### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by maximeseguin**
Add install command for external tools on Debian/Ubuntu distribution

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,8 +8,10 @@ Some external tools need to be present on your system for Micmac to run properly
 - [exiftool](www.sno.phy.queensu.ca/~phil/exiftool) and [exiv2](www.exiv2.org), to read/write image meta-data,
 - [proj4](http://trac.osgeo.org/proj/) for coordinate system conversion.
 
-You can check before-hand that Micmac is able to find those programs by calling the command :
+On Debian/Ubuntu distribution you can easily install this tools by calling this command :
```
Comment (rjanvier) [inline README.md]: this -> these

File: README.md [hunk 2]
```diff
@@ -8,8 +8,10 @@ Some external tools need to be present on your system for Micmac to run properly
...
+`sudo apt-get install make imagemagick libimage-exiftool-perl exiv2 proj-bin qt5-default`
 
+You can check before-hand that Micmac is able to find those programs by calling the command :
```
Comment (rjanvier) [inline README.md]: "command :" -> "command: "

**Comment by rjanvier** [comment]
@maximeseguin do you know how to modify your pull request? Or maybe it's easier for you if I merge your modifications first and we'll push the requested changes afterward?

**Comment by maximeseguin** [comment]
@rjanvier I don't know how to modify my pull request. It will be easier for to push afterward the requested changes.

**Pull request merged**